src/Schema/Directives/CanDirective.php

```diff
@@ -176,10 +176,20 @@ protected function authorize(Gate $gate, $ability, $model, array $arguments): vo
         // should be [modelClassName, additionalArg, additionalArg...]
         array_unshift($arguments, $model);
 
-        if (! $gate->check($ability, $arguments)) {
-            throw new AuthorizationException(
-                "You are not authorized to access {$this->nodeName()}"
-            );
+        if (is_array($ability)) {
```

Comment:
This can be replaced by `Utils::applyEach()`

Reply:
Done. Makes the code look nicer, thanks for suggestion!